plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -444,7 +448,10 @@ public static ColumnMapping timestampColumnMappingUsingSqlTimestamp(TimestampTyp
                 timestampType,
                 (resultSet, columnIndex) -> {
                     Timestamp timestamp = resultSet.getTimestamp(columnIndex);
-                    return toPrestoTimestamp(timestampType, timestamp.toLocalDateTime());
+                    long epochMicros = toPrestoTimestamp(timestampType, timestamp.toLocalDateTime());
+                    verify(epochMicros == round(epochMicros, MAX_SHORT_PRECISION - timestampType.getPrecision()),
+                            "Invalid value of epochMicros for precision %s: %s", timestampType.getPrecision(), epochMicros);
```

Comment:
nit: we put all arguments on one line, or each on separate line

```
verify(
  expr,
 msg.
  arg);
```

or 
```
verify(expr, msg, ar);
```

Reply:
Done!